apps/src/StudioApp.js

```diff
@@ -2140,6 +2145,18 @@ StudioApp.prototype.configureDom = function (config) {
       }
     });
   }
+  // Sends a Statsig event when the Run button is pressed by a signed out user
+  // This is related to the Create Account Button A/B Test; see Jira ticket:
+  // https://codedotorg.atlassian.net/browse/ACQ-1938
+  if (runButton && isSignedOut && isStandaloneProject) {
+    dom.addClickTouchEvent(runButton, () => {
```

Comment:
Is it a problem that the `runButton` will have two click/touch handlers? (I genuinely don't know)

Reply:
I don't know either 😅 @sanchitmalhotra126 do you have any insight here, or is there anyone else we can ask?

Reply:
I'm not sure either honestly 😅 but for safety I might suggest keeping it in one click handler? We can wrap `runClick` with a function that calls the run handler and also fires the event. That way the event will also fire when the key handler above runs.

Reply:
Ok, I think I have a fix for this! Commit: https://github.com/code-dot-org/code-dot-org/pull/59656/commits/d5431a265c4550b8375f1f0bbce80146f55dfd5d